src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -1089,6 +1089,68 @@ internal void SelectAllCategories(object parameter)
             }
         }
 
+        /// <summary>
```

Comment:
@aosyatnik - No. Why cannot we say false to only those categories where Isexpanded is set to true? 
Why we should traverse all the categories here?